test/controllers/tasks_controller_test.rb

```diff
@@ -0,0 +1,15 @@
+require 'foreman_tasks_test_helper'
+
+module ForemanTasks
+  class TasksControllerTest < ActionController::TestCase
+
+    describe ForemanTasks::TasksController do
+      basic_pagination_per_page_test
```

Comment:
I thought it required some more changes to the controller, but it didn't. Added that